File: Programmers/Lv0/문자열정렬하기1.py
def solution(my_string):
    return sorted([int(num) for num in my_string if '0' <= num <= '9'])

# sort메서드는 배열을 정렬하고 None을 반환한다.
# sorted함수는 정렬된 배열을 반환한다.

File: Programmers/Lv0/문자열정렬하기1.cpp
#include <string>
#include <vector>
#include <algorithm>
using namespace std;

vector<int> solution(string my_string) {
    vector<int> answer;
    for (char ch : my_string)
        if (isdigit(ch))
            answer.push_back(static_cast<int>(ch - '0'));
    sort(answer.begin(), answer.end());
    return answer;
}